src/universal/containers/app/MyAccountPage/MyAccountPage.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  Paper
+} from './../../../components';
+import {initialize} from 'redux-form';
+import { connect } from 'react-redux';
+import { updatePersonalInformationAction } from './../../../redux/reducers/auth';
+/*
+import { Link } from 'react-router';
+import { connect } from 'react-redux';
+import { initialize } from 'redux-form';
+import { pushState } from 'redux-router';
+*/
+import PersonalInformationForm from './PersonalInformationForm';
+
+
+@connect(
+  state => ({
+    user: state.auth.user
+  }), {
+    initialize,
+    updatePersonalInformationAction
+  })
+export default class MyAccountPage extends Component {
+  static propTypes = {
+    user: PropTypes.object.isRequired,
+    initialize: PropTypes.func.isRequired,
+    updatePersonalInformationAction: PropTypes.func.isRequired
+  };
+
+  componentWillMount() {
+    const {
+      user: {
+        firstName,
+        lastName,
+        companyName
+      }
+    } = this.props;
+
+    this.props.initialize('PersonalInformationForm', {firstName, lastName, companyName});
```

Comment:
Put the string `'PersonalInformationForm'` inside `PersonalInformationForm` component.

PersonalInformationForm:
```
export const PERSONAL_INFORMATION_FORM_NAME = 'PersonalInformationForm';
```
MyAccountPage:
```
import PersonalInformationForm, {PERSONAL_INFORMATION_FORM_NAME} from './PersonalInformationForm';
....
this.props.initialize(PERSONAL_INFORMATION_FORM_NAME, {firstName, lastName, companyName});
```